packages/e2e-tests/specs/performance/site-editor.test.js

```diff
@@ -52,7 +55,7 @@ describe( 'Site Editor Performance', () => {
 		while ( i-- ) {
 			const startTime = new Date();
 			await page.reload();
-			await page.waitForSelector( '.wp-block' );
+			await page.waitForSelector( '.wp-block', { timeout: 120000 } );
```

Comment:
Why is such a large timeout needed?

Reply:
I see the comment that the screen is taking longer. I'm surprised this takes longer than 30 seconds. :)